### PR TITLE
docs(codes): Windows Support: Display Adjustment Commands

### PR DESCRIPTION
**Pull request by KingCoinless**
### Windows version/build?
Windows 10 Version 1909
### What codes did you test?
C_BRI_UP
C_BRI_DN
C_BRI_MIN
C_BRI_MAX
C_BRI_AUTO
C_BKLT_TOG
C_ASPECT
C_PIP
### How did you test?
Manual use with my Helix split keyboard using the associated ZMK shield on the desktop and in video windows.
### Have you any useful information?
Nothing but `C_BRI_UP` worked whatsoever to change the brightness or perspective settings of my display.
### Has anyone else tested/verified it?
Not as far as I am aware, no.

### CODE REVIEW

File: docs/src/data/hid.js [hunk 1]
```diff
@@ -5182,7 +5182,7 @@ export default [
     ],
     documentation: "https://usb.org/sites/default/files/hut1_2.pdf#page=134",
     os: {
-      windows: null,
+      windows: false,
```
Comment (caksoylar) [inline docs/src/data/hid.js:5185]:
```suggestion
      windows: true,
```
This also seems to work for me, tested on Windows 11 desktop (shows the brightness slider but doesn't affect the external display) and Windows 10 Surface Book (actually sets screen brightness down as expected.

**Review by caksoylar — APPROVED** [review]
Looks good to me based on the code changes and the deploy preview.

I tested all eight on my Surface Book and brightness up/down work well as I noted in the comment. Others don't seem to do anything for me however I haven't tried with a video player application, just on the desktop.

File: docs/src/data/hid.js (hunk 1 above)
Comment (KingCoinless) [reply docs/src/data/hid.js:5185]: Looks good, committing changes. An aside, what version of windows is the Surface Book on? My testing was done on Win 10 1909.

Comment (caksoylar) [reply docs/src/data/hid.js]: Thanks! This is Surface Book Gen 1, on Windows 10 21H2 19044.1415.

**Review by dxmh — APPROVED** [review]

**Pull request merged**